### PR TITLE
Jl production tcr table

### PR DESCRIPTION
**Pull request by dividedsky**
* Adds teachers_classes_refreshrs table
* removes teacher_id from classes table
* removes class_id from refreshrs table
* moves date from refreshrs table to teachers_classes_refreshrs
* updates seed files for new schema

I git pulled from master before pushing. In case there are any merge conflicts, the only files I changed were the one migration file and the three seed files.

### CODE REVIEW

File: client/src/components/classes/CampaignForm.js [hunk 1]
```diff
@@ -1,215 +1,108 @@
...
     justifyContent: 'center',
-    margin: '1rem',
+    margin: '1rem'
+  },
+  cardList: {
+    border: '1px solid blue'
   },
   card: {
-    alignItems: 'center',
+    margin: 20,
```
Comment (szincone) [inline client/src/components/classes/CampaignForm.js]: Maybe add `px` on your sizes at a later date just for readability purposes.

File: client/src/components/classes/CampaignForm.js [hunk 2]
```diff
@@ -1,215 +1,108 @@
...
 
-    }
-    console.log(`after: ${refreshr.date}`);
-    // console.log(date);
+  // fetch class refreshrs on mount
+  useEffect(() => {
+    getRefreshrs();
+  }, []);
 
+  /* putting the axios request here for now just to design the page. it may make more sense to make
```
Comment (szincone) [inline client/src/components/classes/CampaignForm.js]: I agree.

File: client/src/components/classes/CampaignForm.js [hunk 3]
```diff
@@ -1,215 +1,108 @@
...
-      <p>CampaignForm Component</p>
-      {props.refreshrs && props.refreshrs.length && props.refreshrs.map(refreshr => (
-        <div className={props.classes.card} key={refreshr.id} id={refreshr.refreshr_id}>
-          {refreshr.date}
-          <TextField type='date'onChange={(e) => addDate(e.target.value, refreshr.refreshr_id)}/>
-        </div>
-        ))}
-    </Grid>
+      <Grid container className={classes.wrapper}>
+        <Typography variant="h6">Refreshrs(campaign)</Typography>
```
Comment (szincone) [inline client/src/components/classes/CampaignForm.js]: I'm going to have to play w/ this more, it looks like `typography` makes your stuff responsive?

Comment (szincone) [reply client/src/components/classes/CampaignForm.js]: *more responsive

File: client/src/components/classes/notes.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+        question:
+        wrong_answer:
+        right_answer:
+      }
+    ]
+  }
+]
+
+Refreshr =
+Questions =
```
Comment (szincone) [inline client/src/components/classes/notes.js]: Looks like this is still a work in progress (just this file).

File: server/data/migrations/production/20190225171120_teachers_classes_refreshrs.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  function refreshrDropCIdAndDate() {
+    return knex.schema.table('refreshrs', tbl => {
+      tbl.dropColumn('class_id');
+      tbl.dropColumn('date');
+    });
+  }
+
+  function makeTCRTable() {
```
Comment (szincone) [inline server/data/migrations/production/20190225171120_teachers_classes_refreshrs.js]: Good work, this is a pretty complicated schema.

File: client/src/components/classes/CampaignForm.js (hunk 1 above)
Comment (dividedsky) [reply client/src/components/classes/CampaignForm.js]: Yes, good call!

File: client/src/components/classes/CampaignForm.js (hunk 3 above)
Comment (dividedsky) [reply client/src/components/classes/CampaignForm.js]: Yeah I'm not 100% clear all it does myself, but it seems material ui prefers you to use the typography component for text.

**Pull request merged**